qa/suites/rados/standalone/supported-random-distro$

```diff
@@ -0,0 +1 @@
+../basic/supported-random-distro$
```

Comment:
need to move the workloads into a subdir, so they aren't all combined into one job